package.json

```diff
@@ -6,12 +6,15 @@
   "scripts": {
     "start": "node index.js",
     "test": "cross-env NODE_ENV=test mocha ./test/**/*.js",
-    "coverage": "cross-env NODE_ENV=test ./node_modules/istanbul/lib/cli.js cover ./node_modules/mocha/bin/_mocha",
-    "lint": "eslint ."
+    "coverage": "npm run pretest && cross-env NODE_ENV=test ./node_modules/istanbul/lib/cli.js cover ./node_modules/mocha/bin/_mocha && npm run posttest",
+    "pretest": "sequelize --env test db:migrate && sequelize --env test db:seed",
+    "posttest": "sequelize --env test db:migrate:undo:all",
+    "migration": " sequelize db:migrate:undo:all && sequelize db:migrate   "
   },
```

Comment:
You have removed the script for linting on line 10. Can you please add it so we can always lint to clear errors before pushing